Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs

```diff
@@ -667,5 +667,10 @@ public virtual void SceneReady(Scene scene, LoadSceneMode mode)
 
 			}
 		}
-	}
+
+        public static GameObject GetGameObjectByNetworkId(uint id)
+        {
+            return ((NetworkBehavior)Instance.Networker.NetworkObjects[id].AttachedBehavior).gameObject;
```

Comment:
I personally would either check that the id passed in exists in the dictionary or use the `TryGetValue` method.